CONTRIBUTING

```diff
@@ -15,9 +15,13 @@ https://twistedmatrix.com/trac/wiki/ContributingToTwistedLabs
 
 Twisted has a Code of Conduct, available at code_of_conduct.md.
 
-**Warning: pull requests are ignored!** File a ticket at:
+**Warning: pull requests are ignored** unless that have an associated ticket.
```

Comment:
Grammar nitpick:  "unless **they** have an associated ticket", or "pull requests that don't have an associated ticket are ignored"